### PR TITLE
pt-BR Request for Review - Video Introduction to Web Accessibility and W3C Standards

### PR DESCRIPTION
**Pull request by lucasnnobrega**
Created a new file (index.pt-BR.md) and translated the index.md to pt-BR.

When published, it will be available at: https://www.w3.org/WAI/videos/standards-and-benefits/pt-BR

### CODE REVIEW

**Comment by slhenry** [comment]
Thank you Lucas!

See the [Translation Notes](https://github.com/w3c/wai-video-standards-and-benefits#translation-notes). I checked [if the player is available in pt-BR](https://github.com/ableplayer/ableplayer/blob/master/README.md#user-content-supported-languages). It looks like it is not?

If the player is not available in pt-BR, then in your translation, the player user interface words should be in English. (and you do not need another image) So you should **not** do what it says under the first bullet "If the is available in the translated language,", -- **you should follow the instruction under the second bullet: "If the player is not available in the translated language"** :-)

**Comment by slhenry** [comment]
Hi @lucasnnobrega I hope things are going well with you. I'm just checking in on translations and thought I'd send you a reminder. No urgency. :-)

**Comment by lucasnnobrega** [comment]
Hi @slhenry,
I'm sorry for the late reply, I did not understand in the first time, because I thought that because I had the subtitles in Portuguese I needed to translate the player words and add an image.
I will fix to be in perfect to merge.

**Comment by lucasnnobrega** [comment]
Hi @slhenry 
I saw the translated folder inside ableplayer repository and there is a Portuguese translation for the player, but the README.md for the repository is not updated with this latest version. Now I don't know how to procede with this new information. Thanks for all the help.

**Comment by slhenry** [comment]
Thanks for the update. I'll look into it...

**Comment by yatil** [comment]
Hi! We have the translations file in the Theme: https://github.com/w3c/wai-website-theme/tree/master/assets/ableplayer/translations

I’m unsure why it is not taken up, might be because we are using the (standard) language descriptor `pt-BR` while the language file is `pt-br` and thus not matched.

I can only investigate this next week as I’m not working today (evidently).

**Comment by slhenry** [comment]
@yatil 

> language descriptor pt-BR while the language file is pt-br and thus not matched.

Yup, that is the issue. I'll leave it to you to decide which to change...

**Comment by slhenry** [comment]
@lucasnnobrega

Per above, you are correct to have the player terms in pt-BR :-)

Please translate:
* On a W3C server — video (file format: MP4, file size: 28MB), Brazilian portuguese subtitle file (VTT), list of other language files
* Web Accessibility Perspectives - Audio Described (YouTube playlist).
* Web Accessibility Perspectives web page
* Web Accessibility
* [Shadi speaking]
* please send it to
* Translating WAI Resources

Thanks!

**Comment by yatil** [comment]
I have fixed the able player issue (see: https://github.com/ableplayer/ableplayer/pull/385) and committed the fix to our site. It works locally and I’ll now have the preview re-generated.

**Comment by lucasnnobrega** [comment]
Hi @slhenry 
I fixed the last sentences in English and now the code can be merged.

**Comment by yatil** [comment]
Confirming that locally the UI is translated, despite it not being translated in the preview here.

File: index.pt-BR.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+### Traduzir para outros idiomas
+
+Agradecemos as traduções adicionais. Se você traduzir o
+<<<<<<< HEAD
+[arquivo VTT]({{ "/content-images/wai-video-standards-and-benefits/W3C_INTRO_SFHI.vtt" | relative_url }}), por favor enviar para o email <public-wai-translations@w3.org>. Se você talvez deseja traduzir esse site inteiro ou outras, por favor veja [[Traduzindo recursos WAI]](/about/translating/).
+=======
+[arquivo VTT]({{ "/content-images/wai-video-standards-and-benefits/W3C_INTRO_SFHI.vtt" | relative_url }}), please send it to <public-wai-translations@w3.org>. Se você talvez deseja traduzir esse site inteiro ou outras, por favor veja [[Translating WAI Resources]](/about/translating/).
+
+>>>>>>> 434ec78e1a4ae13365a4c8d24eba094aabc4642f
```
Comment (yatil) [inline index.pt-BR.md]: It looks like there was a conflict checked in :-)

Comment (yatil) [reply index.pt-BR.md]: I have fixed this :-)

**Comment by yatil** [comment]
@slhenry The preview is now ready to review: https://deploy-preview-61--wai-video-standards-and-benefits.netlify.com/videos/standards-and-benefits/pt-br

**Comment by slhenry** [comment]
Thanks @lucasnnobrega ! The [preview](https://deploy-preview-61--wai-video-standards-and-benefits.netlify.com/videos/standards-and-benefits/pt-br) now has the video player in pt_BR. Could you redo that last screen shot clip? ([instructions](https://github.com/w3c/wai-video-standards-and-benefits#translation-notes))

I'll send an e-mail to the translators list asking for review now.

**Comment by lucasnnobrega** [comment]
Hi @slhenry!
I redo the last screenshot like your request and committed the change in the repository.
Waiting for further instructions.

**Comment by slhenry** [comment]
Thank you!

Let's give reviewers a few days...

File: index.pt-BR.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+           subtitles="W3C_INTRO_SFHI.ar.vtt|ar,W3C_INTRO_SFHI.de.vtt|de,W3C_INTRO_SFHI.es.vtt|es,W3C_INTRO_SFHI.fr.vtt|fr,W3C_INTRO_SFHI.gu.vtt|gu,W3C_INTRO_SFHI.hi.vtt|hi,W3C_INTRO_SFHI.kok.vtt|kok,W3C_INTRO_SFHI.ko.vtt|ko,W3C_INTRO_SFHI.ml.vtt|ml,W3C_INTRO_SFHI.mr.vtt|mr,W3C_INTRO_SFHI.nl.vtt|nl,W3C_INTRO_SFHI.pt-BR.vtt|pt-BR|default,W3C_INTRO_SFHI.te.vtt|te,W3C_INTRO_SFHI.zh.vtt|zh,W3C_INTRO_SFHI.ja.vtt|ja,W3C_INTRO_SFHI.it.vtt|it,W3C_INTRO_SFHI.hu.vtt|hu,W3C_INTRO_SFHI.el.vtt|el,W3C_INTRO_SFHI.ru.vtt|ru"
+           %}
+
+Esta informação do vídeo está disponível:
+
+- **Em diferentes idiomas** — [Instruções para ver as legendas e os vídeos transcritos](#traducoes) estão abaixo
+- Abaixo com o **[Texto Transcrito com a Descrição das Imagens](#transcript)**
+- [No YouTube](https://www.youtube.com/watch?v=20SHvU2PKsM)
+- No servidor da W3C — [video (file format: MP4, file size: 28MB)](http://media.w3.org/wai/accessibility-intro/intro.mp4), [arquivo de legenda em português Brasileiro (VTT)](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.pt-BR.vtt), [lista de outros arquivos de idioma](http://media.w3.org/wai/accessibility-intro/)
```
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
- No servidor do W3C — [video (formato do arquivo: MP4, tamanho do arquivo: 28MB)](http://media.w3.org/wai/accessibility-intro/intro.mp4), [arquivo de legenda em português Brasileiro (VTT)](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.pt-BR.vtt), [lista de outros arquivos de idioma](http://media.w3.org/wai/accessibility-intro/)
```

File: index.pt-BR.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+ref: /videos/standards-and-benefits/ # Do not change this
+layout: default
+github:
+  repository: w3c/wai-video-standards-and-benefits
+  path: "index.pt-BR.md" # Add the language shortcode to the middle of the filename, for example index.fr.md
+footer:
+  > # Translate all the words below, including "Date:" and "Editor:". Do not change these dates.
+  <p><strong>Data:</strong> Atualizado 21 Fevereiro 2019. Publicado inicialmente 4 December 2017. <a href="./changelog/">Log de mudanças</a>.</p>
+  <p>Diretor do projeto: <a href="https://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Colaboradores: <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>, <a href="https://www.w3.org/People/Brewer/">Judy Brewer</a>, <a href="https://www.w3.org/People/yatil/">Eric Eggert</a>. Gravação e edição do vídeo: Ulrich Grimm, av-design GmbH. Vídeos desenvolvido com o suporte de <a href="https://www.w3.org/WAI/DEV/">WAI-DEV project</a>, co-fundado pela Comissão Europeia. Traduções gerenciadas com o suporte de <a href="https://www.w3.org/WAI/expand-access/">WAI Expanding Access project</a>, fundado pela Ford Foundation.</p>
```
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
  <p>Diretor do projeto: <a href="https://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>. Colaboradores: <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>, <a href="https://www.w3.org/People/Brewer/">Judy Brewer</a>, <a href="https://www.w3.org/People/yatil/">Eric Eggert</a>. Gravação e edição do vídeo: Ulrich Grimm, av-design GmbH. Vídeos desenvolvidos com o suporte da <a href="https://www.w3.org/WAI/DEV/">WAI-DEV project</a>, co-financiado pela Comissão Europeia. Traduções gerenciadas com o suporte da <a href="https://www.w3.org/WAI/expand-access/">WAI Expanding Access project</a>, financiado pela Ford Foundation.</p>
```

File: index.pt-BR.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after #.
+title: "Vídeo de Introdução a Acessibilidade Web e Padrões W3C" # Do not translate "title:". Do translate the text after "title:".
+nav_title: "Vídeo de Introdução" # A short title that is used in the navigation
+lang: pt-BR # Change "en" to the translated language shortcode from https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
+last_updated: 2019-07-23 # Put the date of this translation YYYY-MM-DD (with month in the middle)
+translators:
+  - name: "Lucas Nobrega" # Replace @@ with name, add separate - name lines for every translator
+contributors:
+  - name: "Reinaldo Ferraz" # Replace @@ with name, add separate - name lines for every
```
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
  - name: "Reinaldo Ferraz" # Replace @@ with name, add separate - name lines for every
    - name: "Caroline Burle" # Replace @@ with name, add separate - name lines for every
```

File: index.pt-BR.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+---
+
+{% include video-player.html
+           yt-id="20SHvU2PKsM"
+           path="/content-images/wai-video-standards-and-benefits/"
+           captions="W3C_INTRO_SFHI.vtt|en"
+           subtitles="W3C_INTRO_SFHI.ar.vtt|ar,W3C_INTRO_SFHI.de.vtt|de,W3C_INTRO_SFHI.es.vtt|es,W3C_INTRO_SFHI.fr.vtt|fr,W3C_INTRO_SFHI.gu.vtt|gu,W3C_INTRO_SFHI.hi.vtt|hi,W3C_INTRO_SFHI.kok.vtt|kok,W3C_INTRO_SFHI.ko.vtt|ko,W3C_INTRO_SFHI.ml.vtt|ml,W3C_INTRO_SFHI.mr.vtt|mr,W3C_INTRO_SFHI.nl.vtt|nl,W3C_INTRO_SFHI.pt-BR.vtt|pt-BR|default,W3C_INTRO_SFHI.te.vtt|te,W3C_INTRO_SFHI.zh.vtt|zh,W3C_INTRO_SFHI.ja.vtt|ja,W3C_INTRO_SFHI.it.vtt|it,W3C_INTRO_SFHI.hu.vtt|hu,W3C_INTRO_SFHI.el.vtt|el,W3C_INTRO_SFHI.ru.vtt|ru"
+           %}
+
+Esta informação do vídeo está disponível:
```
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
As informações sobre esse vídeo estão disponíveis:
```

File: index.pt-BR.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+
+- **Em diferentes idiomas** — [Instruções para ver as legendas e os vídeos transcritos](#traducoes) estão abaixo
```
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
- **Em diferentes idiomas** — [Instruções para ver as legendas e os vídeos transcritos](#traduções) estão abaixo
```

File: index.pt-BR.md (hunk 2 above)
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
- No servidor do W3C — [vídeo (file format: MP4, file size: 28MB)](http://media.w3.org/wai/accessibility-intro/intro.mp4), [arquivo de legenda em português Brasileiro (VTT)](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.pt-BR.vtt), [lista de outros arquivos de idioma](http://media.w3.org/wai/accessibility-intro/)
```

File: index.pt-BR.md [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+  "[English captions file (VTT)](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.en.vtt)", with translation of:
+  "[[your language] subtitle file (VTT)](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.[lang code].vtt)".
+  {% endcomment %}
+
+## Mais informações {#moreinfo}
+
+Para mais informações, veja:
+
+- [[**Introdução a Acessibilidade Web**]](/fundamentals/accessibility-intro/)
+- [[**Videos de Perspectivas de Acessiblilidade Web**: Explore o impacto e os benefícios para todo o mundo]](/perspective-videos/) — videos e informação sobre temas específicos de acessibilidade.
```
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
- [[**Vídeos sobre Perspectivas de Acessibilidade Web**: Explore o impacto e os benefícios para todos]](/perspective-videos/) — vídeos e informação sobre temas específicos de acessibilidade.
```

File: index.pt-BR.md [hunk 8]
```diff
@@ -0,0 +1,224 @@
...
+- [**WAI website**](https://www.w3.org/WAI/) — para encontrar uma ampla variedade de recursos sobre diferentes aspectos [padrões](/standards-guidelines/), [educação](/teach-advocate/), [teste/evolução](/test-evaluate/), [gestão de projetos e política](/planning/) na acessibilidade web.
+
+## Permissão para Usar o Vídeo {#permission}
+
+Você pode usar esse vídeo se incluir um _link_ para esta página. Mais informações disponiveis em [[Usando Material do WAI: Permissão para Usar com Reconhecimento]](/about/using-wai-material/).
```
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
Você pode usar esse vídeo se incluir um _link_ para esta página. Mais informações disponíveis em [[Usando Material do WAI: Permissão para Usar com Reconhecimento]](/about/using-wai-material/).
```

File: index.pt-BR.md [hunk 9]
```diff
@@ -0,0 +1,224 @@
...
+    <tr>
+      <td>Navegadores Web, players de mídia e aplicativos também precisam dar suporte a recursos de acessibilidade.</td>
+      <td>Exemplo de navegador da web usado para mostrar o conteúdo da web</td>
+    </tr>
+    <tr>
+      <td>O W3C fornece padrões para ajudar a tornar a Web acessível, reconhecidos internacionalmente por governos e empresas.</td>
+      <td>[Shadi falando]</td>
+    </tr>
+    <tr>
+      <td>O mais conhecido são as Diretrizes de Acessibilidade para Conteúdo Web - WCAG. As WCAG também são o padrão ISO 40500, e adotadas no padrão europeu denominado EN 301 549. Elas são fundamentadas em torno de quarto princípios:</td>
```
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
      <td>Alguns dos mais conhecidos são as Diretrizes de Acessibilidade para Conteúdo Web - WCAG. As WCAG também são o padrão ISO 40500, e adotadas no padrão europeu denominado EN 301 549. Elas são fundamentadas em torno de quarto princípios:</td>
```

File: index.pt-BR.md (hunk 4 above)
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
  - name: "Reinaldo Ferraz" # Replace @@ with name, add separate - name lines for every
  - name: "Caroline Burle" # Replace @@ with name, add separate - name lines for every
```

File: index.pt-BR.md (hunk 2 above)
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
- No servidor do W3C — [vídeo (file format: MP4, file size: 28MB)](http://media.w3.org/wai/accessibility-intro/intro.mp4), [arquivo de legenda em português Brasileiro (VTT)](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.pt-BR.vtt), [lista de outros arquivos de idioma](http://media.w3.org/wai/accessibility-intro/)
```

File: index.pt-BR.md (hunk 7 above)
Comment (cburle) [inline index.pt-BR.md]:
```suggestion
- [[**Vídeos sobre Perspectivas de Acessibilidade na Web**: Explore o impacto e os benefícios para todos]](/perspective-videos/) — vídeos e informações sobre temas específicos de acessibilidade.
```

File: index.pt-BR.md [hunk 10]
```diff
@@ -0,0 +1,219 @@
...
+
+## Permissão para Usar o Vídeo {#permission}
+
+Você pode usar esse vídeo se incluir um _link_ para esta página. Mais informações disponíveis em [[Usando Material da WAI: Permissão para Usar com Atribuição]](/about/using-wai-material/).
+
+## Descrição de Áudio {#audio-description}
+
+Este vídeo não inclui uma descrição de áudio porque as imagens somente reafirmam o áudio e não proveem informação adicional. Neste caso, a descrição do áudio seria mais distrativa do que útil para a maioria das pessoas, incluindo aquelas que não podem ver as imagens. A Transcrição com a Descrição abaixo inclui descrição das imagens do vídeo.
+
+Se deseja exemplos de vídeos com descrição de áudio, acesse [Perspectivas de Acessibilidade Web - Descrição do Audio (lista de reprodução do YouTube)](https://www.youtube.com/watch?v=21yWr7evHTs&list=PLhDEeYUfW02Qo4r2KlzagxZxhYcZADee-). Ou você pode selecionar um dos vídeos [no site Perspectivas de Acessibilidade Web](/perspective-videos/) e selecionar "{% include t.html t='Enable Audio Description' %}" ("<span lang='en'>Enable Audio Description</span>").
```
Comment (brunopulis) [inline index.pt-BR.md:60]: The word audio is missing from orthographic accentuation.

**Comment by shawna-slh** [comment]
I'm so sorry that I lost this. I am publishing it tonight.

Did [comment from @brunopulis](https://github.com/w3c/wai-video-standards-and-benefits/pull/61#pullrequestreview-420582399) get addressed?

If not, please open new pull request on the published resource. (If it's a little grammar fix, I can merge it without additional review. :-)

We will be promoting translations the week of 17 May 2021 along with GAAD - Global Accessibility Awareness Day. We are creating a changelog for this resource in hopes someone will want to update this translation. We also welcome additional translations!

~Shawn

**Pull request closed without merging**